docs/getting-started/quickstart.md

```diff
@@ -39,49 +43,69 @@ The [`run`](https://substreams.streamingfast.io/reference-and-specs/command-line
+- Go compiler
+- Rust compiler
+- [`substreams` CLI](https://substreams.streamingfast.io/reference-and-specs/command-line-interface)
+- StreamingFast authentication key
 
-The manifest defines the name of the module handler and the protocol buffer to use as the data definition for the Substreams module.&#x20;
+### 1. Create Substreams manifest
 
-The commands demonstrate how Substreams modules work by using the selected blockchain and instructing the compute engine to execute the `map_basic_eth` or `map_basic_sol` Rust functions, also called "_module handlers_".
+To create a "Substreams module", you must first create the manifest file. Use the example manifest provided in the [`substreams-ethereum-tutorial`](https://github.com/streamingfast/substreams-ethereum-tutorial)repository on GitHub as a guide. This example manifest includes the minimal required fields to demonstrate the core values that you must provide.
 
-Use the [`run`](https://substreams.streamingfast.io/reference-and-specs/command-line-interface#run) command for the Ethereum example by using:
+To use the example manifest, copy and paste it into a new file named substreams.yaml. Save this file in the root directory of your "Substreams module." You can find the example manifest in the [official GitHub repository for `substreams-ethereum-tutorial`](https://github.com/streamingfast/substreams-ethereum-tutorial).
 
-{% code title="etheruem substreams run" overflow="wrap" %}
-```bash
-substreams run -e mainnet.eth.streamingfast.io:443 substreams-ethereum-tutorial.yaml map_basic_eth --start-block 10000001 --stop-block +1
-```
-{% endcode %}
+```yaml
+specVersion: v0.1.0
+package:
+  name: 'substreams_ethereum_tutorial'
+  version: v0.1.0
 
-Use the [`run`](https://substreams.streamingfast.io/reference-and-specs/command-line-interface#run) command for the Solana example by using:
+protobuf:
+  files:
+    - basicexample.proto
+  importPaths:
+    - ./proto
 
-{% code title="solana substreams run" overflow="wrap" %}
-```bash
-substreams run -e mainnet.sol.streamingfast.io:443 substreams-solana-tutorial.yaml map_basic_sol
+imports:
+  eth: https://github.com/streamingfast/sf-ethereum/releases/download/v0.10.2/ethereum-v0.10.4.spkg
+
+binaries:
+  default:
+    type: wasm/rust-v1
+    file: target/wasm32-unknown-unknown/release/substreams_ethereum_tutorial.wasm
+
+modules:
+  - name: map_basic_eth
+    kind: map
+    inputs:
+      - source: sf.ethereum.type.v2.Block
+    output:
+      type: proto:eth.basicexample.v1.BasicExampleProtoData
 ```
-{% endcode %}
 
-Different blockchains have specific requirements for their data definitions. The code in the module handlers needs to be updated to match the expectations of the separate blockchains.
+### 2. Create Rust configuration file
```

Comment:
```suggestion
### 2. Create Rust manifest file
```

Reply:
Updated. 

(All the literature I've seen refers to this as a Rust configuration file.)

Reply:
Ok interesting, could you link me some of them, going to take a look, maybe I'm wrong :)

Reply:
Ha! It looks like where I had read this was wrong! I just found the manifest reference in the Rust docs! My bad! I'll have to find where I read this. I know I did, I researched what to call it when writing this a while back.

Very interesting. Sorry for any confusion!!

https://doc.rust-lang.org/cargo/reference/manifest.html